bmrg-flow/values.yaml

```diff
@@ -172,8 +172,8 @@ monitoring:
 
 # Eventing related values
 eventing:
-  enabled: false
```

Comment:
Are we planning to remove the non-eventing path and rely only on the eventing?

Reply:
Nope, eventing will be controlled from the Settings screen in the web app.